module/os/linux/zfs/vdev_object_store.c

```diff
@@ -683,6 +707,14 @@ agent_resume(void *arg)
 	VERIFY0(agent_open_pool(vd, vos,
 	    vdev_object_store_open_mode(spa_mode(vd->vdev_spa))));
 
+	if ((ret = agent_resume_state_check(vd)) != 0) {
+		zfs_dbgmsg("agent resume failed, stated changed");
```

Comment:
```suggestion
		zfs_dbgmsg("agent resume failed, state changed");
```
Or perhaps more specifically: `uberblock changed`